ow_lander/scripts/unstow_action_server.py

```diff
@@ -58,7 +58,8 @@ def _update_motion(self):
         print("Unstow arm activity started")
         goal = self._interface.move_arm.get_current_pose().pose
```

Comment:
Why is goal written to here if it immediately written to on the next line?

Reply:
same response.. I am pretty sure the first call is not needed. I will remove it